src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.jsx

```diff
@@ -39,8 +39,7 @@ const organizationNavigation = [
       {
         path: `${pathPrefix}/auth/`,
         title: t('Auth'),
-        show: ({organization, access, features}) =>
-          features.has('sso-basic') && access.has('org:admin'),
+        show: ({organization, access, features}) => access.has('org:admin'),
```

Comment:
Do you need to unpack `organization` and `features` if you don't use them? :bike:

Reply:
nope